features/handle-delete.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Create(Article)
+  We want to handle Delete(Note) activities in the Inbox
+
+  Background:
+    Given an Actor "Person(Bob)"
+    And we follow "Bob"
+    And the request is accepted
+    And a "Accept(Follow(Bob))" Activity "Accept" by "Bob"
+    And "Bob" sends "Accept" to the Inbox
+    And "Accept" is in our Inbox
+    Given an Actor "Person(Alice)"
```

Comment:
Remember we ideally only want 1 `Given`, `When` and `Then` then use `And` to link together the steps. I know that then means theres a lot of `Ands` here